_posts/zh/newsletters/2019-05-07-newsletter.md

```diff
@@ -0,0 +1,94 @@
+
+    [输出脚本描述符][Output script descriptors]是 Pieter Wuille 开发的一种新语言，用于简洁地描述 scriptPubKeys 以便钱包准确知道应该扫描什么。最终目标是让 Bitcoin Core 的钱包包含一个描述所有脚本的简单描述符列表——对于大多数用户来说，这个列表可能只有一个描述符，但支持未来升级和高级用例（包括多重签名和硬件签名）的广泛灵活性；请参见 PR [#15487][Bitcoin Core #15487] 和 [#15764][Bitcoin Core #15764] 以了解实现此目标的工作。然而，为了让用户和项目开发人员在对钱包进行根本性更改之前积累使用描述符的经验，0.18 版本更新了现有 RPC 并添加了新的 RPC 来处理描述符。带有描述符支持的现有 RPC 包括 `scantxoutset`、`importmulti`、`getaddressinfo` 和 `listunspent`。新的 RPC 包括 `deriveaddresses` 和 `getdescriptorinfo`。
+
+  - **<!--basic-hardware-signer-support-through-independent-tool-->***通过独立工具提供基本的硬件签名支持：* 虽然与 0.18 分开发布，但仍然是 Bitcoin Core 项目的一部分，[硬件钱包交互][HWI](HWI)工具允许熟悉命令行的用户使用 Bitcoin Core 与几种流行的硬件钱包一起工作。内部，该工具大量使用 PSBT 和输出脚本描述符，使其可以与支持这些接口的其他钱包（例如 [Wasabi 钱包的实验性支持][wasabi hwi]）集成。已经开始了将 HWI 更直接地与主要的 Bitcoin Core 工具集成并为其构建图形界面的工作。
+
+  - **<!--new-wallet-tool-->***新钱包工具：* 在 `bitcoind` 和其他比特币程序旁边是一个新的 `bitcoin-wallet` 工具。这个命令行工具目前仅允许用户创建新钱包或对现有钱包进行一些基本检查，但计划在后续版本中为该工具添加更多功能。
+
+  - **<!--new-architecture-and-new-ubuntu-snap-package-->***新架构和新的 Ubuntu Snap 包：* 这是第一个为 RISC-V CPU 架构的 Linux 提供[预编译二进制文件][bitcoincore.org download]的版本。对于 Ubuntu 和兼容系统的用户，此版本还提供了一个 [Snap 包][Snap package]，取代了以前版本中更新的 PPA。这些 RISC-V 和 Snap 包都包括由多个 Bitcoin Core 贡献者[确定性构建和签名][gitian sigs]的二进制文件。
+
+    {% comment %}<!--
+    152 Tests and QA
+    74 Docs
+    65 wallet
+    55 RPCs and other APIs
+    51 GUI
+    47 Build system
+    43 Misc
+    17 p2p and network code
+    13 Platform support
+    9 block and tx handling
+    1 mining
+    1 consensus
+    -->{% endcomment %}
+
+  - **<!--numerous-testing-and-quality-assurance-qa-changes-->***大量的测试和质量保证 (QA) 更改：* [发布说明][0.18 notes]列出了 Bitcoin Core 发布经理认为重要的与 0.18 相关的所有 PR，分为十二类。尽管重要性和分类标准有些任意，并且 PR 数量不一定与所做工作的量相关，但我们认为值得注意的是，发布说明中的“测试和 QA”部分的 PR 数量是其他任何类别的两倍以上。我们不常在这个 Newsletter 中写测试内容——测试很少成为新闻，除非出现问题——所以我们想借此机会提醒读者，测试仍然是 Bitcoin Core 开发的一个活跃且重要的部分。
+
+  - **<!--plan-to-switch-to-bech32-receiving-addresses-by-default-->***计划默认切换到 bech32 接收地址：* 如 [Newsletter #40][] 的*新闻*部分所述，发布说明宣布项目计划在下一个主要版本（0.19，[预计在2019年11月左右发布][0.19 release schedule]）或之后的版本（预计约一年后发布）中默认切换到 bech32 发送地址。较早的日期是当前目标。
+
+- **<!--proposal-for-support-of-schnorr-signatures-and-taproot-script-commitments-->****Schnorr 签名和 Taproot 脚本承诺支持的提案：** Pieter Wuille [发布][tap post]了一个使用 Schnorr 签名的 [Taproot BIP 提案][taproot]和一个用于 Taproot 限制的 [Tapscript BIP 提案][tapscript]。还提供了一个[参考实现][tap ref]，它在大约 520 行代码中（不包括将添加到 [libsecp256k1][] 库中的与 Schnorr 签名相关的更改；请参阅之前发布的 [bip-schnorr][] 以获取更多信息）进行了所有共识更改。
+
+  提案发布的时间太晚，无法让我们在本期 Newsletter 中提供详细描述，尽管我们确实修改了本期 Newsletter 中的其他一些文字以反映提案的发布。我们计划在下周提供完整的报道。
+
+## Bech32 发送支持
+
+*第 8 周，共 24 周。在 2019 年 8 月 24 日 segwit 软分叉锁定的第二周年纪念日之前，Optech Newsletter 将包含这个每周部分，提供帮助开发者和组织实现 bech32 发送支持——支付本机 segwit 地址的能力的信息。这[不需要你自己实现 segwit][bech32 series]，但它确实允许你支付的人访问 segwit 的所有好处。*
+
+{% comment %}<!-- weekly reminder for harding: check Bech32 Adoption
+wiki page for changes -->{% endcomment %}
+
+{% include specials/bech32/zh/08-upgradability.md %}
```

Comment:
@bittomhan can you add this zh/08-upgradability.md reference to the zh/bech32 post page as well?

Reply:
Thank you! I have completed the modifications. In the future, I will perform a rebase on each submission sequentially.

Reply:
@bittomhan this doesnt appear to be in place. Still referencing the en version:
https://github.com/bitcoinops/bitcoinops.github.io/pull/1754/files#diff-d7e4457adb7e4fd45a53f9219da2f3c80db818edfb9e191f0d4e9295af002807L73